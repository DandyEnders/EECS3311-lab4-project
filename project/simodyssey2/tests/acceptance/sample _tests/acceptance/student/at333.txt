move(W)
move(NW)
move(W)
pass
move(NW)
move(N)
wormhole
move(NW)
pass
move(NE)
move(W)
move(SE)
test(29,32,36,38,40)
move(NW)
move(NE)
move(E)
move(S)
play
move(W)
play
move(NW)
