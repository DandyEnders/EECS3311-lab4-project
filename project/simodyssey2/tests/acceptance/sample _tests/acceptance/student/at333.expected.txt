  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(W)
  state:0.1, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.2, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.3, error
  Negative on that request:no mission in progress.
->pass
  state:0.4, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.5, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.6, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.7, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.8, error
  Negative on that request:no mission in progress.
->pass
  state:0.9, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.10, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.11, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.12, error
  Negative on that request:no mission in progress.
->test(29,32,36,38,40)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->[1,B],[-4,*],-,-
    [1,3]->[2,J],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[3,J],[4,A],-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],-,-,-
    [2,3]->[5,M],-,-,-
    [2,4]->[6,A],[7,A],[-11,W],-
    [2,5]->[8,A],[9,B],-,-
    [3,1]->[10,A],[-2,Y],-,-
    [3,2]->[11,M],[-7,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[12,A],-,-,-
    [3,5]->[13,A],-,-,-
    [4,1]->[14,A],-,-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->[15,A],[16,M],-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],-,-,-
    [5,1]->[-6,Y],-,-,-
    [5,2]->[17,J],-,-,-
    [5,3]->[-9,Y],-,-,-
    [5,4]->[18,J],[19,B],[20,A],-
    [5,5]->[-5,W],-,-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [2,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [4,A]->turns_left:0
    [5,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [6,A]->turns_left:2
    [7,A]->turns_left:2
    [8,A]->turns_left:1
    [9,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [10,A]->turns_left:2
    [11,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:2
    [12,A]->turns_left:2
    [13,A]->turns_left:0
    [14,A]->turns_left:1
    [15,A]->turns_left:1
    [16,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [17,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [18,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [19,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [20,A]->turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   B*--   J---   ----   JA--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   M---   AAW-   AB--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AY--   M*--   O---   A---   A---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    A---   *---   AM--   ----   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y---   J---   Y---   JBA-   W---   
->move(NW)
  state:2.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[5,5,2]
    [4,A]:[1,5,2]->[1,1,1]
    [5,M]:[2,3,1]->[3,3,2]
    [13,A]:[3,5,1]->[3,4,2]
  Sectors:
    [1,1]->[4,A],-,-,-
    [1,2]->[1,B],[-4,*],-,-
    [1,3]->[2,J],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[3,J],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],-,-,-
    [2,3]->-,-,-,-
    [2,4]->[6,A],[7,A],[-11,W],-
    [2,5]->[8,A],[9,B],-,-
    [3,1]->[10,A],[-2,Y],-,-
    [3,2]->[11,M],[-7,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[12,A],[13,A],-,-
    [3,5]->-,-,-,-
    [4,1]->[14,A],-,-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->[15,A],[16,M],-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],-,-,-
    [5,1]->[-6,Y],-,-,-
    [5,2]->[17,J],-,-,-
    [5,3]->[-9,Y],-,-,-
    [5,4]->[18,J],[19,B],[20,A],-
    [5,5]->[-5,W],[0,E],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [2,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [4,A]->turns_left:0
    [6,A]->turns_left:1
    [7,A]->turns_left:1
    [8,A]->turns_left:0
    [9,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [10,A]->turns_left:1
    [11,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [12,A]->turns_left:1
    [13,A]->turns_left:2
    [14,A]->turns_left:0
    [15,A]->turns_left:0
    [16,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [17,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [18,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [19,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [20,A]->turns_left:1
  Deaths This Turn:
    [5,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:N/A,
      Malevolent got devoured by blackhole (id: -1) at Sector:3:3
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    A---   B*--   J---   ----   J---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   ----   AAW-   AB--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AY--   M*--   O---   AA--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    A---   *---   AM--   ----   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y---   J---   Y---   JBA-   WE--   
->move(NE)
  state:3.0, mode:test, ok
  Movement:
    [0,E]:[5,5,2]->[4,1,2]
    [1,B]:[1,2,1]->[5,1,2]
    [2,J]:[1,3,1]->[2,2,2]
    [4,A]:[1,1,1]->[1,5,2]
      destroyed [3,J] at [1,5,1]
    [8,A]:[2,5,1]->[3,4,3]
    [9,B]:[2,5,2]->[1,5,1]
    [14,A]:[4,1,1]->[5,1,3]
      destroyed [1,B] at [5,1,2]
    [15,A]:[4,3,1]->[5,2,2]
      destroyed [17,J] at [5,2,1]
    [16,M]:[4,3,2]->[3,2,3]
    [19,B]:[5,4,2]->[4,5,2]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->-,[-4,*],-,-
    [1,3]->-,-,-,-
    [1,4]->-,-,-,-
    [1,5]->[9,B],[4,A],-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],[2,J],-,-
    [2,3]->-,-,-,-
    [2,4]->[6,A],[7,A],[-11,W],-
    [2,5]->-,-,-,-
    [3,1]->[10,A],[-2,Y],-,-
    [3,2]->[11,M],[-7,*],[16,M],-
    [3,3]->[-1,O],-,-,-
    [3,4]->[12,A],[13,A],[8,A],-
    [3,5]->-,-,-,-
    [4,1]->-,[0,E],-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->-,-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],[19,B],-,-
    [5,1]->[-6,Y],-,[14,A],-
    [5,2]->-,[15,A],-,-
    [5,3]->[-9,Y],-,-,-
    [5,4]->[18,J],-,[20,A],-
    [5,5]->[-5,W],-,-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:1/3, life:3/3, landed?:F
    [2,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:1/2, turns_left:0
    [4,A]->turns_left:0
    [6,A]->turns_left:0
    [7,A]->turns_left:0
    [8,A]->turns_left:1
    [9,B]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [10,A]->turns_left:0
    [11,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [12,A]->turns_left:0
    [13,A]->turns_left:1
    [14,A]->turns_left:0
    [15,A]->turns_left:0
    [16,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [18,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
    [19,B]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [20,A]->turns_left:0
  Deaths This Turn:
    [3,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 4) at Sector:1:5
    [1,B]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:N/A,
      Benign got destroyed by asteroid (id: 14) at Sector:5:1
    [17,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:N/A,
      Janitaur got destroyed by asteroid (id: 15) at Sector:5:2
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   -*--   ----   ----   BA--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *J--   ----   AAW-   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    AY--   M*M-   O---   AAA-   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    -E--   *---   ----   ----   WB--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y-A-   -A--   Y---   J-A-   W---   
->move(E)
  state:4.0, mode:test, ok
  Movement:
    [0,E]:[4,1,2]->[4,2,2]
    [2,J]:[2,2,2]->[1,1,1]
    [4,A]:[1,5,2]->[5,5,2]
    [6,A]:[2,4,1]->[1,3,1]
    [7,A]:[2,4,2]->[1,4,1]
    [10,A]:[3,1,1]->[2,5,1]
    [11,M]:[3,2,1]->[3,3,2]
    [12,A]:[3,4,1]->[3,3,2]
    [14,A]:[5,1,3]->[5,2,1]
    [15,A]:[5,2,2]->[5,3,2]
    [18,J]:[5,4,1]->[1,4,2]
      destroyed [7,A] at [1,4,1]
    [20,A]:[5,4,3]->[5,3,3]
  Sectors:
    [1,1]->[2,J],-,-,-
    [1,2]->-,[-4,*],-,-
    [1,3]->[6,A],-,-,-
    [1,4]->-,[18,J],-,-
    [1,5]->[9,B],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],-,-,-
    [2,3]->-,-,-,-
    [2,4]->-,-,[-11,W],-
    [2,5]->[10,A],-,-,-
    [3,1]->-,[-2,Y],-,-
    [3,2]->-,[-7,*],[16,M],-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,[13,A],[8,A],-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->[-3,*],[0,E],-,-
    [4,3]->-,-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],[19,B],-,-
    [5,1]->[-6,Y],-,-,-
    [5,2]->[14,A],-,-,-
    [5,3]->[-9,Y],[15,A],[20,A],-
    [5,4]->-,-,-,-
    [5,5]->[-5,W],[4,A],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [2,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:0/2, turns_left:2
    [4,A]->turns_left:2
    [6,A]->turns_left:0
    [8,A]->turns_left:0
    [9,B]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [10,A]->turns_left:1
    [13,A]->turns_left:0
    [14,A]->turns_left:2
    [15,A]->turns_left:2
    [16,M]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:0
    [18,J]->fuel:4/5, load:1/2, actions_left_until_reproduction:1/2, turns_left:1
    [19,B]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [20,A]->turns_left:2
  Deaths This Turn:
    [11,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:N/A,
      Malevolent got devoured by blackhole (id: -1) at Sector:3:3
    [12,A]->turns_left:N/A,
      Asteroid got devoured by blackhole (id: -1) at Sector:3:3
    [7,A]->turns_left:N/A,
      Asteroid got imploded by janitaur (id: 18) at Sector:1:4
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    J---   -*--   A---   -J--   B---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   ----   --W-   A---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   -*M-   O---   -AA-   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   *E--   ----   ----   WB--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y---   A---   YAA-   ----   WA--   
->move(S)
  state:5.0, mode:test, ok
  Movement:
    [0,E]:[4,2,2]->[5,2,2]
    [6,A]:[1,3,1]->[2,4,1]
    [8,A]:[3,4,3]->[2,3,1]
    [9,B]:[1,5,1]->[5,1,2]
      reproduced [21,B] at [5,1,3]
    [13,A]:[3,4,2]->[3,3,2]
    [16,M]:[3,2,3]->[2,3,2]
      reproduced [22,M] at [2,3,3]
    [19,B]:[4,5,2]->[2,4,2]
      reproduced [23,B] at [2,4,4]
  Sectors:
    [1,1]->[2,J],-,-,-
    [1,2]->-,[-4,*],-,-
    [1,3]->-,-,-,-
    [1,4]->-,[18,J],-,-
    [1,5]->-,-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],-,-,-
    [2,3]->[8,A],[16,M],[22,M],-
    [2,4]->[6,A],[19,B],[-11,W],[23,B]
    [2,5]->[10,A],-,-,-
    [3,1]->-,[-2,Y],-,-
    [3,2]->-,[-7,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->-,-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],-,-,-
    [5,1]->[-6,Y],[9,B],[21,B],-
    [5,2]->[14,A],[0,E],-,-
    [5,3]->[-9,Y],[15,A],[20,A],-
    [5,4]->-,-,-,-
    [5,5]->[-5,W],[4,A],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [2,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:0/2, turns_left:1
    [4,A]->turns_left:1
    [6,A]->turns_left:1
    [8,A]->turns_left:0
    [9,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [10,A]->turns_left:0
    [14,A]->turns_left:1
    [15,A]->turns_left:1
    [16,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:2
    [18,J]->fuel:4/5, load:1/2, actions_left_until_reproduction:1/2, turns_left:0
    [19,B]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:2
    [20,A]->turns_left:1
    [21,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:2
    [22,M]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [23,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
  Deaths This Turn:
    [13,A]->turns_left:N/A,
      Asteroid got devoured by blackhole (id: -1) at Sector:3:3
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    J---   -*--   ----   -J--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   AMM-   ABWB   A---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   -*--   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   *---   ----   ----   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    YBB-   AE--   YAA-   ----   WA--   
->play
  state:5.1, mode:test, error
  To start a new mission, please abort the current one first.
->move(W)
  state:6.0, mode:test, ok
  Movement:
    [0,E]:[5,2,2]->[5,1,4]
    [8,A]:[2,3,1]->[2,2,2]
    [10,A]:[2,5,1]->[1,5,1]
    [18,J]:[1,4,2]->[1,3,1]
    [22,M]:[2,3,3]->[3,4,1]
  Sectors:
    [1,1]->[2,J],-,-,-
    [1,2]->-,[-4,*],-,-
    [1,3]->[18,J],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[10,A],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],[8,A],-,-
    [2,3]->-,[16,M],-,-
    [2,4]->[6,A],[19,B],[-11,W],[23,B]
    [2,5]->-,-,-,-
    [3,1]->-,[-2,Y],-,-
    [3,2]->-,[-7,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[22,M],-,-,-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->-,-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-8,W],-,-,-
    [5,1]->[-6,Y],[9,B],[21,B],[0,E]
    [5,2]->[14,A],-,-,-
    [5,3]->[-9,Y],[15,A],[20,A],-
    [5,4]->-,-,-,-
    [5,5]->[-5,W],[4,A],-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [2,J]->fuel:4/5, load:0/2, actions_left_until_reproduction:0/2, turns_left:0
    [4,A]->turns_left:0
    [6,A]->turns_left:0
    [8,A]->turns_left:1
    [9,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [10,A]->turns_left:2
    [14,A]->turns_left:0
    [15,A]->turns_left:0
    [16,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:1
    [18,J]->fuel:3/5, load:1/2, actions_left_until_reproduction:0/2, turns_left:2
    [19,B]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:1
    [20,A]->turns_left:0
    [21,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:1
    [22,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:1
    [23,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    J---   -*--   J---   ----   A---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *A--   -M--   ABWB   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   -*--   O---   M---   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   *---   ----   ----   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    YBBE   A---   YAA-   ----   WA--   
->play
  state:6.1, mode:test, error
  To start a new mission, please abort the current one first.
->move(NW)
  state:7.0, mode:test, ok
  Movement:
    [0,E]:[5,1,4]->[4,5,2]
    [2,J]:[1,1,1]->[2,5,1]
      reproduced [24,J] at [2,5,2]
    [4,A]:[5,5,2]->[4,4,1]
    [6,A]:[2,4,1]->[3,5,1]
    [9,B]:[5,1,2]->[4,1,1]
    [14,A]:[5,2,1]->[1,1,1]
    [15,A]:[5,3,2]->[5,2,1]
    [20,A]:[5,3,3]->[1,4,1]
    [23,B]:[2,4,4]->[5,4,1]
  Sectors:
    [1,1]->[14,A],-,-,-
    [1,2]->-,[-4,*],-,-
    [1,3]->[18,J],-,-,-
    [1,4]->[20,A],-,-,-
    [1,5]->[10,A],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-10,*],[8,A],-,-
    [2,3]->-,[16,M],-,-
    [2,4]->-,[19,B],[-11,W],-
    [2,5]->[2,J],[24,J],-,-
    [3,1]->-,[-2,Y],-,-
    [3,2]->-,[-7,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[22,M],-,-,-
    [3,5]->[6,A],-,-,-
    [4,1]->[9,B],-,-,-
    [4,2]->[-3,*],-,-,-
    [4,3]->-,-,-,-
    [4,4]->[4,A],-,-,-
    [4,5]->[-8,W],[0,E],-,-
    [5,1]->[-6,Y],-,[21,B],-
    [5,2]->[15,A],-,-,-
    [5,3]->[-9,Y],-,-,-
    [5,4]->[23,B],-,-,-
    [5,5]->[-5,W],-,-,-
  Descriptions:
    [-11,W]->
    [-10,*]->Luminosity:5
    [-9,Y]->Luminosity:2
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [2,J]->fuel:3/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:2
    [4,A]->turns_left:0
    [6,A]->turns_left:1
    [8,A]->turns_left:0
    [9,B]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:2
    [10,A]->turns_left:1
    [14,A]->turns_left:1
    [15,A]->turns_left:1
    [16,M]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:0
    [18,J]->fuel:3/5, load:1/2, actions_left_until_reproduction:0/2, turns_left:1
    [19,B]->fuel:2/3, actions_left_until_reproduction:1/1, turns_left:0
    [20,A]->turns_left:0
    [21,B]->fuel:3/3, actions_left_until_reproduction:1/1, turns_left:0
    [22,M]->fuel:2/3, actions_left_until_reproduction:0/1, turns_left:0
    [23,B]->fuel:3/3, actions_left_until_reproduction:0/1, turns_left:1
    [24,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    A---   -*--   J---   A---   A---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *A--   -M--   -BW-   JJ--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   -*--   O---   M---   A---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    B---   *---   ----   A---   WE--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y-B-   A---   Y---   B---   W---   
